### PR TITLE
Fix : Asc 20383- Added video loading indicator,  ASC 20389 - Fixed Video post is doubled when click like

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/components/Social/PostList/index.tsx [hunk 1]
```diff
@@ -188,99 +178,61 @@ export default function PostList({
...
+      if (isLike) {
+        await removePostReaction(postId, 'like');
       } else {
-        dispatch(updateByPostId({ postId: post.postId, postDetail: post }));
+        await addPostReaction(postId, 'like');
       }
-
-      await addPostReaction(postId, 'like');
+    } catch (error) {
+      console.error('Error:', error);
```
Comment (bonanaaaaaa) [inline src/components/Social/PostList/index.tsx:223]: remove this line

File: src/hooks/useTimeDifference.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import { useMemo } from 'react';
+
+dayjs.extend(relativeTime);
+
+export const useTimeDifference = (timestamp: string): string => {
```
Comment (bonanaaaaaa) [inline src/hooks/useTimeDifference.ts:7]: 👍

File: src/components/Social/PostList/index.tsx (hunk 1 above)
Comment (NaingAmity) [reply src/components/Social/PostList/index.tsx]: removed Done !

**Review by bonanaaaaaa — APPROVED** [review]
LGTM

**Pull request merged**